contrib/net/http/roundtripper.go

```diff
@@ -29,7 +34,8 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 		tracer.SpanType(ext.SpanTypeHTTP),
 		tracer.ResourceName(defaultResourceName),
 		tracer.Tag(ext.HTTPMethod, req.Method),
-		tracer.Tag(ext.HTTPURL, req.URL.Path),
+		tracer.Tag(ext.HTTPURL, req.URL.String()),
+		tracer.Tag("http.path", req.URL.Path),
```

Comment:
Can you please revert this change? We don't want to do this because it may contain sensitive information such as authentication tokens, login credentials, etc.